prow/config.yaml

```diff
@@ -2931,7 +2931,7 @@ periodics:
       - name: GOOGLE_APPLICATION_CREDENTIALS
         value: /etc/service-account/service-account.json
       - name: USER
-        value: prow
+        value: jenkins
```

Comment:
I think the better way is to move the presubmit job to prow so that they can be consistent. Presubmit uses ssh key for user `jenkins` though.